server/migrations/helpers.test.js

```diff
@@ -15,57 +16,63 @@ describe(calculatePreferences, () => {
         config: 'foo',
       },
     }
+
     expect(calculatePreferences(user)).toEqual({
-      config: 'foo',
-      sort: 0,
-      star: [
-        { site: 'CA', starredSubjects: [] },
-        { site: 'LA', starredSubjects: [] },
+      siteSubjects: [
+        { site: 'CA', starredSubjects: [], completedSubjects: [] },
+        { site: 'LA', starredSubjects: [], completedSubjects: [] },
       ],
-      complete: [{ site: 'CA', completedSubjects: [] }],
+      updatedPreferences: { config: 'foo', sort: 0 },
     })
   })
 
   it('retains user preferences if it is using the new data structure', () => {
     const user = {
+      access: ['CA', 'LA', 'YA', 'ProNET'],
+
       preferences: {
         config: '636d1b784723201176bf0bda',
         sort: 0,
-        star: [
-          {
-            site: 'LA',
-            starredSubjects: [],
-          },
-        ],
-        complete: [
-          {
-            site: 'YA',
-            completedSubjects: ['YA01508'],
-          },
-          {
-            site: 'ProNET',
-            completedSubjects: ['P6'],
-          },
-        ],
       },
+      siteSubjects: [
+        {
+          site: 'LA',
+          starredSubjects: [],
+          completedSubjects: [],
+        },
+        {
+          site: 'YA',
+          completedSubjects: ['YA01508'],
+          starredSubjects: [],
```

Comment:
can we add starred subjects too so we can ensure we're adding them properly?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/241/commits/bf0423c6c7f4d2b24d8c720cb39b4accdaccbbc7

Reply:
there are no starred subjects in this test in the commit referenced.

Reply:
I think this comment is resolved. This file has gone through many iterations.